pkg/dependencies/factory.go

```diff
@@ -1324,14 +1324,14 @@ func (f *Factory) WithGitOpsFlux(clusterConfig *v1alpha1.Cluster, fluxConfig *v1
 
 func (f *Factory) WithPackageInstaller(spec *cluster.Spec, packagesLocation, kubeConfig string) *Factory {
```

Comment:
Might be worth changing the method names as well if you are changing the interface